install/kubernetes/Makefile

```diff
@@ -100,14 +57,7 @@ lint:
 docs:
 	$(QUIET)$(HELM_DOCS)
 
-check-docker-images:
-	$(QUIET)\
-         HUBBLE_UI_VERSION=$(HUBBLE_UI_VERSION) \
-         HUBBLE_UI_BACKEND_VERSION=$(HUBBLE_UI_BACKEND_VERSION) \
-         MANAGED_ETCD_VERSION=$(MANAGED_ETCD_VERSION) \
-         ETCD_VERSION=$(ETCD_VERSION) \
-         NODEINIT_VERSION=$(NODEINIT_VERSION) \
-         CERTGEN_VERSION=$(CERTGEN_VERSION) \
-         ../../contrib/release/check-docker-images.sh "v$(VERSION)"
-
-.PHONY: all check-docker-images clean docs lint update-versions
+check-docker-images: # Check whether docker images are available for the current version.
+	$(QUIET)$(ROOT_DIR)/contrib/release/check-docker-images.sh "$(CILIUM_VERSION)"
+
+.PHONY: all check-docker-images clean docs lint update-chart update-versions
```

Comment:
does `cilium/values.yaml` need to be added as a phony target, or is it enough to just have `update-versions` here?

Reply:
Every target that is *not* a file should be there.